ynr/apps/people/models.py

```diff
@@ -373,6 +373,18 @@ def get_edit_url(self, request=None):
             return path
         return request.build_absolute_uri(path)
 
+    def queued_image(self):
+        queued_image = self.queuedimage_set.filter(decision="undecided").first()
```

Comment:
You can make this a little more concise by using:
```
return self.queuedimage_set.filter(decision="undecided").first()
```
As `first()` will return `None` if there is no object matching your query ([docs](https://docs.djangoproject.com/en/4.0/ref/models/querysets/#first))